test/controllers/old_node_controller_test.rb

```diff
@@ -188,29 +188,31 @@ def test_current_version
   # authorised.
   def test_redact_node_unauthorised
     do_redact_node(nodes(:node_with_versions_v3),
-                   redactions(:example))
+                   create(:redaction))
     assert_response :unauthorized, "should need to be authenticated to redact."
   end
 
   ##
   # test the redaction of an old version of a node, while being
   # authorised as a normal user.
   def test_redact_node_normal_user
-    basic_authorization(users(:public_user).email, "test")
+    user = create(:user)
+    basic_authorization(user.email, "test")
 
     do_redact_node(nodes(:node_with_versions_v3),
-                   redactions(:example))
+                   create(:redaction, :user => user))
```

Comment:
There's no need for the redaction to be owned by the same user that is trying use it right? I mean the old code doesn't seem to correlate them like this?